src/FilterList.js

```diff
@@ -2,16 +2,84 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import {ResultPropTypes} from './ResultPropTypes'
 
-const FilterList = ({filteredResults, resultsMessage, ResultsHeaderClass, ResultElementClass}) => {
-  const filteredElements = filteredResults.map((result) => result.element)
-
-  return (
-    <div>
-      <h4>{resultsMessage}</h4>
-      { filteredElements.length && ResultsHeaderClass && <ResultsHeaderClass /> }
-      { filteredElements.map((element, index) => <div key={index}><ResultElementClass {...element}/></div>) }
-    </div>
+function dynamicSort(property, sortState) {
```

Comment:
We use arrow notation for functions, so instead of `function dynamicSort(property, sortState) {` we do `const dynamicSort = (property, sortState) => {`. Change this also in the returned funciton and in `titleCase` below.